stubs/pygit2/pygit2/__init__.pyi

```diff
@@ -0,0 +1,423 @@
+from _typeshed import AnyStr_co, SupportsAllComparisons
+from collections.abc import Callable
+from os import PathLike
+
+from . import enums
+from ._build import __version__ as __version__
+from ._pygit2 import (
```

Comment:
The runtime just does `from ._pygit2 import *`, why not do that here?

Reply:
Originally I kept the star imports, but then mypy started to complain about wrong types of `settings` (it seems unable to distinguish the module `settings` and the global variable `settings` for example), and stubtest also wasn't happy with all those re-exported names either, so I had to spell out the names to basically help the checkers properly resolve the names.